src/Common/Arg.fs

```diff
@@ -47,82 +47,82 @@ type ArgParser() =
       sbuf.ToString()
 
 
-    static member ParsePartial(cursor,argv,argSpecs:seq<ArgInfo>,?other,?usageText) =
```

Comment:
These changes were all to address some signature file/implementation argument name mismatch warnings.